strelka/strelka.cwl

```diff
@@ -9,7 +9,7 @@ requirements:
         coresMin: 8
         ramMin: 4000
 arguments:
-    [ { valueFrom: $(runtime.cores), position: 1 },
+    [ { valueFrom: $(inputs.cpu_num), position: 1 },
```

Comment:
As I understand the issue, the problem is with LSF scheduling the jobs with 8 cores requested. This change will only modify the number passed to the strelka command.  Instead of modifying the number of cpus strelka expects, don't you want to modify the number of CPUs requested? See line9 for the resource requirements used when scheduling the LSF job.  

Changing the number of CPUs that strelka thinks it has available will result in twice the run time as before.